src/main/scripts/org/reaktivity/specification/nukleus/http/streams/rfc7230/message.format/request.headers.too.long/server/source.rpt

```diff
@@ -98,14 +85,8 @@ read [0x00 0x00 0x00 0x02]
 read ${sourceOutputEstId}
 read [0x00]
 read (byte:lengthByte)
-read /HTTP\/1.1 400 Bad Request.*\r\n/
+read /HTTP\/1.1 431 Request Header Fields Too Large\r\n/
```

Comment:
Recommend relaxing to pattern match against any 4xx status code with any description.